config-example.py

```diff
@@ -0,0 +1,2 @@
+client_id = "YOUR CLIENT ID HERE"
+client_secret = "YOUR CLIENT SECRET HERE"
```

Comment:
Can you add a line break at the end of this file?
Also, we usually use single-quotes for strings in Python most of the time. Can you fix all of them among your codes?

Ref: https://github.com/osu-mist/style-guides/blob/master/Python.md#quoting-strings